packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts

```diff
@@ -46,10 +46,6 @@ export class W3mConnectingWcView extends LitElement {
 
   // -- Render -------------------------------------------- //
   public override render() {
-    if (!this.wallet) {
-      return html`<w3m-connecting-wc-qrcode></w3m-connecting-wc-qrcode>`
-    }
-
```

Comment:
why remove this?

Reply:
Its redundant check because you have multiple conditions with the same behaviour such as https://github.com/reown-com/appkit/blob/a28acbde989a47ce44245449faa82f9790515e2a/packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts#L110
This always leads to `w3m-connecting-wc-qrcode` component but its confusing as you have multiple entry points as it was